sympy/printing/latex.py

```diff
@@ -212,12 +213,15 @@ def __init__(self, settings=None):
             "rj": r"\mathrm{j}",
             "tj": r"\text{j}",
         }
-        try:
-            self._settings['imaginary_unit_latex'] = \
-                imaginary_unit_table[self._settings['imaginary_unit']]
-        except KeyError:
-            self._settings['imaginary_unit_latex'] = \
-                self._settings['imaginary_unit']
+        imag_unit = self._settings['imaginary_unit']
+        self._settings['imaginary_unit_latex'] = imaginary_unit_table.get(imag_unit, imag_unit)
+
+        diff_operator_table = {
```

Comment:
You should add the default value here, probably both None and "d". There will be situations where one would like to specify that.